ecommerce_extensions/settings/production.py

```diff
@@ -3,3 +3,28 @@
 """
 
 from __future__ import unicode_literals
+
+try:
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+except ImportError:
+    sentry_sdk = DjangoIntegration = None
+
+# Sentry Integration
```

Comment:
I think this comment can be placed before the try and except.

Reply:
I add another comment before try-except to be more understandable in case we need to add code between those code sections(import and integration)